roles/pgbackrest_srv/tasks/ssh_keys.yml

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
This code duplicates roles/pgbackrest/tasks/ssh_keys.yml

I think it should be one pgbackrest role and not two roles with the same code